src/SmartComponents/AdvisoryDetail/AdvisoryDetail.js

```diff
@@ -51,7 +51,7 @@ const AdvisoryDetail = ({ match }) => {
                 breadcrumbs={[
                     {
                         title: intl.formatMessage(messages.titlesPatchAdvisories),
-                        to: paths.advisories.to,
+                        to: paths.path,
```

Comment:
`paths` is an array in Router.js. This looks like a mistake that you are treating it as an object here.